xwiki-platform-core/xwiki-platform-notifications/xwiki-platform-notifications-filters/xwiki-platform-notifications-filters-watch/src/main/java/org/xwiki/notifications/filters/watch/internal/DefaultWatchedEntitiesConfiguration.java

```diff
@@ -94,12 +91,6 @@ public AutomaticWatchMode getAutomaticWatchMode(DocumentReference user)
             return AutomaticWatchMode.valueOf((String) value);
         }
 
-        // Fallback to the value of the Watchlist
```

Comment:
Note that this fallback is removed because we handle that in the `WatchListObjectsRemovalTaskConsumer` by actually setting a new automatic watch xobject if it's missing before removing the WatchListClass xobject. Now since it's done in a TaskConsumer there might be a few delay before it's getting into account leading to the miss of a few filters creation. IMO it's acceptable as it's not something that should happen a lot.